### DISCUSSION TITLE
zephyrus g14 volume control issues

### DISCUSSION THREAD

**Discussion by ac101m** [abskmj/notes#7] [General]
The volume control fix instructions are a partial fix, but not a complete one. The g14 has two sets of speakers; tweeters behind two small grilles on the left and right below the keyboard, and two woofers on the underside. After a plain install, volume control works only for the tweeters. This makes sounds seem "muddy" at lower volumes, but not quieter. With the fix from your guide, the tweeters are silent but the woofers are controlled, giving the impression that the problem is fixed but causing you to lose a lot of detail :(

You can test for this by playing a really high pitched sound [like this](https://www.youtube.com/watch?v=Lya_l8pO0P4). If the tweeters are working it should be loud and obnoxious, otherwise it will be very quiet. You can also try covering up the tweeters with your thumbs to check if that's where the sound is coming from.

I have found similar instructions for getting volume control working on the g14 in other distros, but they all have this same issue. I think maybe nobody has noticed? I don't imagine there are many of us running Linux on these machines!

I should also mention I'm running Ubuntu 20.04 and not PopOS, but I'm fairly sure there shouldn't be any major differences. I am currently hunting for a solution but don't know enough about pulseaudio configuration yet to really understand what I am doing. Either way I thought I'd drop an issue here to let you know about the problem if you weren't already aware.

**Comment by abskmj** [comment]
@ac101m Hello, it is possible. I'm now using Windows 10 as my primary driver now because of the Citrix receiver that I need for my work,

**Reply by abskmj** [reply]
@ac101m I just tried the new pop os 21.04 beta version on the laptop and both the speakers, tweeters work
